nucleo.cwl

```diff
@@ -0,0 +1,810 @@
+class: Workflow
+cwlVersion: v1.0
+id: nucleo
+doc: >-
+  This workflow takes a READ1 and READ2 fastq.gz file generated for MSK-ACCESS
+  assay and generated four different Binary Alignment Map file along with
+  alignment metrics for each.
+label: nucleo
+$namespaces:
+  s: 'https://schema.org/'
+  sbg: 'https://www.sevenbridges.com/'
+inputs:
+  - id: reference_sequence
+    type: File
+    doc: >-
+      Reference sequence file.  Please include ".fai", "^.dict", ".amb" , ".sa",
+      ".bwt", ".pac", ".ann" as  secondary files if they are not present in the
+      same location as the  ".fasta" file
+    secondaryFiles:
+      - .amb
+      - .fai
+      - .sa
+      - ^.dict
+      - .ann
+      - .bwt
+      - .pac
+    'sbg:x': 0
+    'sbg:y': 1281.9375
+  - id: gatk_base_recalibrator_known_sites
+    type:
+      type: array
+      items: File
+      inputBinding:
+        prefix: '--known-sites'
+    secondaryFiles:
+      - .idx
+    'sbg:x': 0
```

Comment:
I can add the github action for removing these x and y tags if we need it

Reply:
sure, please make sure it is done only when it is in development branch, not before that